src/components/Pages/Collection/Page.js

```diff
@@ -33,8 +32,7 @@ CollectionPage.propTypes = {
   collection: shapes.GenericCollection,
   wip: shapes.WIP,
   err: shapes.Error,
-  t: PropTypes.func.isRequired,
-  renderUnit: PropTypes.func.isRequired,
+  t: PropTypes.func.isRequired
```

Comment:
I see renderUnit is still parameter. Why did you remove it here?

Reply:
removed